client/transaction-pool/api/src/lib.rs

```diff
@@ -30,6 +30,8 @@ use sp_runtime::{
 };
 use std::{collections::HashMap, hash::Hash, pin::Pin, sync::Arc};
 
+const LOG_TARGET: &str = "txpool";
```

Comment:
Should this be "txpool-api" or the fact that is a separate nested crate doesn't matter?
I know you left the name you found. My comment just wants to be a reflection.

Reply:
I hesitated for a second. But I think noone wants to do `LOG_TARGET="txpool,txpool-api,txpool-tests"`. IMO all logs should go under single target.

Reply:
We can name it `txpool::api`. If you then enable `txpool`, it will still print every target which is prefixed with `txpool`

Reply:
```suggestion
const LOG_TARGET: &str = "txpool::api";
```